manuskript/exporter/pandoc/plainText.py

```diff
@@ -18,7 +18,7 @@ class markdown(abstractPlainText):
 
 class reST(abstractPlainText):
     name = "reST"
-    description = qApp.translate("Export", """reStructuredText is a lightweight markup language...""")
```

Comment:
I think it would be an improvement to remove the ellipses "..." and end the statement with a period.